src/main/java/com/forms/formswebapp/form/FormController.java

```diff
@@ -20,4 +21,10 @@ public ResponseEntity<Void> createForm(@RequestBody FormRequestDto formRequestDt
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
+    @GetMapping("/{link}")
+    public ResponseEntity<FormResponseDto> getFormByLink(@PathVariable String link) {
+        formFacade.getFormByLink(link);
```

Comment:
Czy celowo aktualnie nie jest nic zwracane w body?

Reply:
Andrzej mi mówił, żeby na razie link sam zostawić. W przyszłości ma zwracać całą ankietę.

Reply:
Chodziło mi o to, że na razie zwracany będzie sam link oprócz reszty pół które były w FormRequest, niejasno się wyraziłem 😅
Ale na razie i tak to nie ma znacznia bo ten response nie będzie nigdzie w tej iteracji używany